Source/Core/Core/IOS/ES/Formats.cpp

```diff
@@ -295,11 +295,16 @@ u16 TMDReader::GetGroupId() const
 
 DiscIO::Region TMDReader::GetRegion() const
 {
+  if (!IsChannel(GetTitleId()))
```

Comment:
I'm not sure if this is the exact condition that determines whether the TMD's region field is reliable or just zero, but it seems close enough based on my testing. It would make sense if titles that aren't launchable from the Wii Menu don't have any region set. Notably, discs always seem to have the TMD's region field set to 0 and I don't know why, but that is not a problem for us since VolumeWii can get the region in a way that doesn't use the TMD.

Reply:
I guess it's good enough for now, but that might a candidate for a source-code comment.